python/ray/dashboard/modules/node/node_head.py

```diff
@@ -170,104 +166,126 @@ def get_internal_states(self):
             "head_node_registration_time_s": self._head_node_registration_time_s,
             "registered_nodes": len(DataSource.nodes),
             "registered_agents": len(DataSource.agents),
-            "node_update_count": self._node_update_cnt,
             "module_lifetime_s": time.time() - self._module_start_time,
         }
 
-    async def _get_nodes(self):
-        """Read the client table.
+    async def _subscribe_for_node_updates(self) -> AsyncGenerator[dict, None]:
```

Comment:
why is it `[dict, None]`

Reply:
`AsyncGenerator[YieldType, SendType]`. SendType is what the caller returns to the generator, like this

```
value: YieldType = await agen.asend(send_type)
```

we don't have that, so None.